backend/tracim_backend/lib/webdav/design.py

```diff
@@ -124,6 +126,16 @@
     'copy' : 'Item copied',
 }
 
+THREAD_MESSAGE = """
+  <p>
+    {posting_time},{comment_owner} wrote :
+  </p>
+  <p>
+    {comment_content}
+  </p>
+
+"""
```

Comment:
@basilelegal, the way to show indented messages is:

- to write "the xxx, basile wrote:"
- to indent the content

In your MR, you put introduction message (the xxx, basile wrote:) and the user message at same indentation level, which is wrong.

Example taken from an existing email:

```
Le 14/12/2018 à 14:27, michel a écrit :
    Bonjour Damien,
    [...]
```

Reply:
Done